src/helpers/wrapPageElement.js

```diff
@@ -10,20 +10,27 @@ const wrapPageElement = ({ element, props }) => {
       ? props.data.markdownRemark
       : null;
 
-  const isHomePage = props.path === '/';
+  const getTitleFromPath = (path) => {
```

Comment:
If we have `react-helmet` we should be able to just add this to the `archived.js` file